pkg/apis/core/validation/seed_test.go

```diff
@@ -733,128 +748,141 @@ var _ = Describe("Seed Validation Tests", func() {
-				zoneName := "a"
-				incorrectZoneName := "b"
-				seed.Spec.Provider.Zones = []string{zoneName}
-				seed.Spec.Settings = &core.SeedSettings{
-					LoadBalancerServices: &core.SeedSettingLoadBalancerServices{
-						Zones: []core.SeedSettingLoadBalancerServicesZones{
-							{
-								Name: incorrectZoneName,
-								Annotations: map[string]string{
-									"nospecialchars^=@":      "bar",
-									"cantendwithadash-":      "bar",
-									"only/one/slash":         "bar",
-									strings.Repeat("a", 254): "bar",
-								},
-								ExternalTrafficPolicy: &policy,
-							},
-							{
-								Name: incorrectZoneName,
-							},
+					errorList := ValidateSeed(seed)
+
+					Expect(errorList).To(BeEmpty())
+				})
+				It("should allow both prober and probe settings to be set, if equal", func() {
+					seed.Spec.Settings = &core.SeedSettings{
+						DependencyWatchdog: &core.SeedSettingDependencyWatchdog{
+							Prober: &core.SeedSettingDependencyWatchdogProber{Enabled: true},
+							Probe:  &core.SeedSettingDependencyWatchdogProbe{Enabled: true},
 						},
-					},
-				}
+					}
 
-				errorList := ValidateSeed(seed)
+					errorList := ValidateSeed(seed)
 
-				Expect(errorList).To(ConsistOf(
-					PointTo(MatchFields(IgnoreExtras, Fields{
-						"Type":  Equal(field.ErrorTypeForbidden),
-						"Field": Equal("spec.settings.loadBalancerServices.zones"),
-					})),
-					PointTo(MatchFields(IgnoreExtras, Fields{
-						"Type":  Equal(field.ErrorTypeInvalid),
-						"Field": Equal("spec.settings.loadBalancerServices.zones[0].annotations"),
-					})),
-					PointTo(MatchFields(IgnoreExtras, Fields{
-						"Type":  Equal(field.ErrorTypeInvalid),
-						"Field": Equal("spec.settings.loadBalancerServices.zones[0].annotations"),
-					})),
-					PointTo(MatchFields(IgnoreExtras, Fields{
-						"Type":  Equal(field.ErrorTypeInvalid),
-						"Field": Equal("spec.settings.loadBalancerServices.zones[0].annotations"),
-					})),
-					PointTo(MatchFields(IgnoreExtras, Fields{
-						"Type":  Equal(field.ErrorTypeInvalid),
-						"Field": Equal("spec.settings.loadBalancerServices.zones[0].annotations"),
-					})),
-					PointTo(MatchFields(IgnoreExtras, Fields{
-						"Type":  Equal(field.ErrorTypeNotSupported),
-						"Field": Equal("spec.settings.loadBalancerServices.zones[0].externalTrafficPolicy"),
-					})),
-					PointTo(MatchFields(IgnoreExtras, Fields{
-						"Type":  Equal(field.ErrorTypeNotFound),
-						"Field": Equal("spec.settings.loadBalancerServices.zones[0].name"),
-					})),
-					PointTo(MatchFields(IgnoreExtras, Fields{
-						"Type":  Equal(field.ErrorTypeNotFound),
-						"Field": Equal("spec.settings.loadBalancerServices.zones[1].name"),
-					})),
-					PointTo(MatchFields(IgnoreExtras, Fields{
-						"Type":  Equal(field.ErrorTypeDuplicate),
-						"Field": Equal("spec.settings.loadBalancerServices.zones[1].name"),
-					})),
-				))
+					Expect(errorList).To(BeEmpty())
+				})
```

Comment:
What is the purpose of these newly added tests in the light of the unchanged validation code?